src/PointwiseFunctions/AnalyticData/GrMhd/CcsnCollapse.hpp

```diff
@@ -9,14 +9,28 @@
 #include <string>
 
 #include "DataStructures/Tensor/TypeAliases.hpp"
+#include "IO/Connectivity.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/CheckH5.hpp"
+#include "IO/H5/EosTable.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/Helpers.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/SourceArchive.hpp"
+#include "IO/H5/Version.hpp"
+#include "IO/H5/Wrappers.hpp"
```

Comment:
I don't think you need any of these includes in the hpp because the cpp file reads in the data, but the class doesn't store the h5 file, it stores the contents. Phrased differently, the hpp doesn't use any of the h5 stuff, so it's only needed in the cpp. Putting it in the cpp will help keep compile times down. In the cpp you only need
```cpp
#include "IO/H5/AccessType.hpp"
#include "IO/H5/EosTable.hpp"
#include "IO/H5/File.hpp"
```